_posts/2016-01-20-set-up-and-run-hubot-in-non-heroku-server.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Set up and run Hubot without using Heroku"
+date: 2016-01-20 11:16:00
+categories: ["hubot", "linux"]
+author: "mauro-oto"
+---
+
+[Hubot](https://hubot.github.com) makes it incredibly easy to setup on a Heroku
+server, by taking advantage of its Procfile. Simply running
```

Comment:
@mauro-oto "by taking advantage of its Procfile" could be "by taking advantage of its Procfile support"?